smoketests/tests/servers.py

```diff
@@ -15,11 +15,11 @@ def test_servers(self):
         self.assertRegex(servers, re.compile(r"^\s*testnet\.spacetimedb\.com\s+https\s+testnet\s*$", re.M))
         self.assertRegex(servers, re.compile(r"^\s*\*\*\*\s+127\.0\.0\.1:3000\s+http\s+localhost\s*$", re.M))
 
-        out = self.spacetime("server", "fingerprint", "127.0.0.1:3000", "-f")
+        out = self.spacetime("server", "fingerprint", "-s", "http://127.0.0.1:3000", "-f")
```

Comment:
instead of `http://127.0.0.1:3000` I would recommend just using `local`

Reply:
```suggestion
        out = self.spacetime("server", "fingerprint", "-s", "local", "-f")
```

Reply:
mm... this is testing the `fingerprint` subcommand specifically, that it accepts using urls instead of names. I'm gonna leave it as is.